pkg/kyverno/apply/command.go

```diff
@@ -159,20 +159,14 @@ func Command() *cobra.Command {
 				return sanitizederror.NewWithError(fmt.Sprintf("require policy"), err)
 			}
 
-			policies, errors := common.GetPolicies(policyPaths)
-			if len(policies) == 0 {
-				if len(errors) > 0 {
-					return sanitizederror.NewWithErrors("failed to read policies", errors)
-				}
-
-				return sanitizederror.New(fmt.Sprintf("no policies found in paths %v", policyPaths))
+			if (len(policyPaths) > 0 && policyPaths[0] == "-") && len(resourcePaths) > 0 && resourcePaths[0] == "-" {
+				return sanitizederror.NewWithError("pass either policy or resource using pipe", err)
```

Comment:
Please update message to:

`a stdin pipe can be used for either policies or resources, not both`